src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -69,6 +74,9 @@ protected void checkChimericInstances(QAReport report, String clsName) throws Ex
             if (isEscaped(rle)) {
                 continue;
             }
+            if (skipList.inSkipList(rle)) {
```

Comment:
Since the variable is a skip list, maybe the method could be called “contains” instead of “inSkipList”

Reply:
Good idea, I renamed it to `containsInstanceDbId`

Reply:
Thanks!  I’m just wondering since this method takes a GKInstance as its parameter, maybe you could have the name “containsInstance” or change the method to take a long value and pass in “rle.getDbId()”.

Reply:
Just wondering about this.